distributions/application/pom.xml

```diff
@@ -4,12 +4,12 @@
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>replication</groupId>
-        <artifactId>replication</artifactId>
+        <groupId>replication.distributions</groupId>
+        <artifactId>distributions</artifactId>
         <version>0.3.0-SNAPSHOT</version>
     </parent>
-    <artifactId>application</artifactId>
```

Comment:
:question: What is the goal of this module? Is it trying to produce a jar or a zipped distribution? Right now it produces a jar, some scripts, some keystores, and an application.yml file. 

Normally the application.yml is bundled into the spring boot jar (as this provides the defaults)

Reply:
Goal is to just create the spring boot jar. Updated pom to include resources/yml in the jar for defaults.